src/rollup-boost/rollup_boost_launcher.star

```diff
@@ -28,17 +28,27 @@ def launch(
 
     service = plan.add_service(ROLLUP_BOOST_SERVICE_NAME, config)
 
-    return config
+    return service
```

Comment:
what's the diff here? I saw that on other files, the pattern seem to be returning the config instead of service

Reply:
The difference here is that service is the `Service` object that contains networking info (e.g. `ip_address`), while config is the build config
https://docs.kurtosis.com/api-reference/starlark-reference/service
https://docs.kurtosis.com/api-reference/starlark-reference/service-config

I think for el, cl launcher they also return `service` but wrapped into a separate struct `el_context`, `cl_context`
https://github.com/ethpandaops/optimism-package/blob/5b085c892aa02448af21e6e50b11b169a879c05f/src/el/op-geth/op_geth_launcher.star#L145